src/app/ui/editor/brush_preview.cpp

```diff
@@ -262,6 +260,8 @@ void BrushPreview::show(const gfx::Point& screenPos)
           if (showPreview)
             showPreviewWithEdges = true;
           m_type = BRUSH_BOUNDARIES;
+          if (tilemapMode == TilemapMode::Tiles)
+            m_type |= CROSSHAIR;
```

Comment:
This should be valid for `app::gen::BrushPreview::EDGES` case too to avoid this bug:

![edges-only](https://github.com/aseprite/aseprite/assets/39654/6b0cf18b-62c8-4429-9bfe-d574ab311625)

My recommended change, remove these lines:

```suggestion
```

And add after the `switch`:

```c++
  // For tiles as we show the edges of the tile, we add the crosshair
  // in the mouse position as a helper.
  if (m_type & BRUSH_BOUNDARIES &&
      tilemapMode == TilemapMode::Tiles) {
    m_type |= CROSSHAIR;
  }
```